Sources/Workspace/ToolsVersionWriter.swift

```diff
@@ -19,7 +19,7 @@ import Utility
 ///   - path: The path of the package.
 ///   - version: The version to write.
 public func writeToolsVersion(at path: AbsolutePath, version: ToolsVersion, fs: inout FileSystem) throws {
-    let file = path.appending(component: ToolsVersion.toolsVersionFileName)
+    let file = Manifest.versionSpecificManifest(path: path, fileSystem: fs)
```

Comment:
I think we should always update the main manifest, so please revert this change.